doc/htmldoc/conf.py

```diff
@@ -25,6 +25,7 @@
 import re
 import pip
 import subprocess
```

Comment:
Is this still needed with the new line 34 in place?

Reply:
Removed